core/insertion_marker_manager.js

```diff
@@ -10,42 +10,48 @@
  */
 'use strict';
 
-goog.provide('Blockly.InsertionMarkerManager');
-
-goog.require('Blockly.blockAnimations');
-goog.require('Blockly.ComponentManager');
-goog.require('Blockly.connectionTypes');
-goog.require('Blockly.Events');
-goog.require('Blockly.internalConstants');
-
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.RenderedConnection');
-goog.requireType('Blockly.utils.Coordinate');
-goog.requireType('Blockly.WorkspaceSvg');
+goog.module('Blockly.InsertionMarkerManager');
+goog.module.declareLegacyNamespace();
+
+// TODO(#5073): Add Blockly require after fixing circular dependency.
+// goog.require('Blockly');
+const blockAnimations = goog.require('Blockly.blockAnimations');
+const constants = goog.require('Blockly.constants');
+const ComponentManager = goog.require('Blockly.ComponentManager');
+const connectionTypes = goog.require('Blockly.connectionTypes');
+const Events = goog.require('Blockly.Events');
+const internalConstants = goog.require('Blockly.internalConstants');
+
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
```

Comment:
These lines need to be disabled for eslint :/
Also the requires are currently not sorted correctly

Reply:
Fixed.